sdk/client.ts

```diff
@@ -258,7 +302,7 @@ class Client {
   /**
    * Generates a new authorization token and attaches it to every API.
    */
-  private async updateBearerToken(): Promise<void> {
+  private async attachBearerToken(): Promise<void> {
     const token: string = await this.getBearerToken()
     this.apis.map((api) => (api.accessToken = token))
   }
```

Comment:
[Info] I had to rename this cause we wanted to have an `update**Token` method for bearer tokens.